tests/sweep_framework/sweeps/max_pool2d/short/max_pool2d_short_sweep.py

```diff
@@ -0,0 +1,191 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional, Tuple, List
+import itertools
+import random
+import torch
+import math
+
+import ttnn
+from ttnn.operations.conv2d import determine_parallel_config, create_sharded_memory_config_from_parallel_config
+
+from tests.ttnn.utils_for_testing import check_with_pcc, start_measuring_time, stop_measuring_time
+from models.utility_functions import torch_random
+
+
+parameters = {
+    "max_pool2d_short_sweep_suite": {
+        "dtype": [ttnn.bfloat16, ttnn.bfloat8_b],
+        "input_specs": [
+            [1, 128, 112, 112, 2, 2, 2, 2, 0, 0, 1, 1, False],
```

Comment:
please add a comment line specifying list of parameter names.

Reply:
Done.